client/rest-high-level/src/test/java/org/elasticsearch/client/DataFrameRequestConvertersTests.java

```diff
@@ -66,14 +66,17 @@ public void testPutDataFrameTransform() throws IOException {
         PutDataFrameTransformRequest putRequest = new PutDataFrameTransformRequest(
                 DataFrameTransformConfigTests.randomDataFrameTransformConfig());
         Request request = DataFrameRequestConverters.putDataFrameTransform(putRequest);
-
+        assertFalse(request.getParameters().containsKey("defer_validation"));
```

Comment:
I'd rewrite it to:
assertThat(request.getParameters(), not(hasKey("defer_validation"));

This has the advantage of spitting better message when the assertion fails.